Terminal.Gui/Core.cs

```diff
@@ -1471,7 +1563,7 @@ public override bool ProcessKey (KeyEvent keyEvent)
 				return true;
 			case Key.CursorLeft:
 			case Key.CursorUp:
-			case Key.BackTab:
+			case Key.BackTab | Key.ShiftMask:
```

Comment:
I suspect that this is a problem on Windows - on Unix, this should be returning Key.BackTab, so I suspect the driver on Windows should clear "ShiftMask" in this case?

Reply:
start review

Reply:
I will.

Reply:
Done